Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -60,7 +60,7 @@ plugin. Restart kube-dns to ensure it is managed by Cilium.
 
 .. code:: bash
 
-     kubectl -n kube-system delete pod -l k8s-app: kube-dns
+     kubectl -n kube-system delete pod -l k8s-app=kube-dns
```

Comment:
@ackerman80 please sign off your commit

Reply:
@ackerman80 the commit is still not signed-off: 
<img width="1397" alt="screen shot 2019-02-12 at 5 31 51 pm" src="https://user-images.githubusercontent.com/3858271/52680080-18846480-2eec-11e9-83fe-8c291e527b9e.png">

Reply:
signed